pyke/kepextract.py

```diff
@@ -579,6 +594,10 @@ def kepextract_main():
                         type=str)
     parser.add_argument('--bkg', action='store_true',
                         help='Subtract background from data?')
+    parser.add_argument('--psfphot', action='store_true',
+                        help='Perform PSF Photometry with 2D Gaussian?')
+    parser.add_argument('--moments', action='store_true',
+                        help='Perform sample moments photometry?')
```

Comment:
How slow is is to compute the centroids using the moments?  I suspect (but don't know) that it is pretty fast.  For K2 it is really necessary to have some information of a star's position for subsequent processing (detrending) steps, so I suggest we always compute the moments by default and not make this an option, or users will be confused as to why they can't run `kepdetrend` after `kepextract` using default options!